### PR TITLE
Solving Issue #771 - Add processing version for coding challenge 117 - seven segment display

### PR DESCRIPTION
**Pull request by fernand0aguilar**
Add Code for Challenge 117: Seven Segment Display #771

Processing version for the seven segment code made on code challenge number 117

### CODE REVIEW

File: CodingChallenges/CC_117_7_Segment_Display/CC_117_7_Segment_Display.pde [hunk 1]
```diff
@@ -0,0 +1,51 @@
+int[] nums = {0x7E, 0x30, 0x6D, 0x79, 0x33, 0x5B, 0x5F, 0x70, 0x7F, 0x7B};
```
Comment (shiffman) [inline CodingChallenges/CC_117_7_Segment_Display/CC_117_7_Segment_Display.pde]: Can you add the usual header stuff here? Check other coding challenges for an example? Thank you!!!

Comment (fernand0aguilar) [reply CodingChallenges/CC_117_7_Segment_Display/CC_117_7_Segment_Display.pde]: I just added the usual header stuff. 
Thanks for all Dan.

**Comment by fernand0aguilar** [comment]
ps: this is the pull request number #777 
this brings luck

**Comment by shiffman** [comment]
Thank you!

**Pull request merged**